tesseract_process_managers/examples/memory_usage_example.cpp

```diff
@@ -0,0 +1,145 @@
+
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <fstream>
+#include <chrono>
+#include <unistd.h>
+#include <string>
+TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
+#include <tesseract_environment/core/environment.h>
+#include <tesseract_environment/ofkt/ofkt_state_solver.h>
+#include <tesseract_command_language/command_language.h>
+#include <freespace_example_program.h>
```

Comment:
Is this correct?

Reply:
Copy past content, I will remove unused headers.

Reply:
Oh, I see now what you were talking about. I will change to use quotes so it changes he search behavior.